src/patternfly/components/TextInputGroup/text-input-group-text-input.hbs

```diff
@@ -2,6 +2,9 @@
   type="text"
   value="{{text-input-group--value}}"
```

Comment:
You could do `{{#if text-input-group--IsDisabled}}Disabled{{else}}{{text-input-group--value}}{{/if}}`